README.md

```diff
@@ -132,25 +132,47 @@ Data is expected in the following format for the `create` scope:
     email: 'fake_email@example.com', // user's email
     firstName: 'First', // user's first name
     lastName: 'Last' // user's last name
+    zipCode: '20009' // user's zip code (optional)
   }
 }
 ```
 
+##### user options
+
+| User Options | Description       | Required |
+| ------------ | ----------------- | -------- |
+| `email`      | user's email      | yes      |
+| `firstName`  | user's first name | yes      |
+| `lastName`   | user's last name  | yes      |
+| `zipCode`    | user's ZipCode    | no       |
+
 #### `update` scope
 
 Data is expected in the following format for the `update` scope:
 
 ```javascript
 {
   user: {
-    id: 1, // user's id in API
+    id: 1, // user's id in the API
   },
   utilityCredential: {
-    id: 2 // utility credential id in API
+    id: 2 // utility credential id in the API
   }
 }
 ```
 
+##### user options
+
+| User Options | Description          | Required |
+| ------------ | -------------------- | -------- |
+| `id `        | user's id in the API | yes      |
+
+##### utility credential options
+
+| Utility Credential Options | Description                      | Required |
+| -------------------------- | -------------------------------- | -------- |
+| `id `                      | utility credential id in the API | yes      |
+
```

Comment:
Seems unnecessary, but I wanted to be consistent with how I'm defining these options in the `create` scope ^^